### DISCUSSION TITLE
Got Plane Parts, What Now?

### DISCUSSION THREAD

**Discussion by AerospaceGuy** [iNavFlight/inav#10257] [Q&A]
I've been wanting to build a model plane from scratch for a while now and recently got the chance, I watched a few stuff and got some info from ai on what parts I would need and got a f405 speedybee flight controller, some servos, a motor, a remote controller, a receiver, and a battery. I own a 3D printer and some foam board and have a good knowledge of aerospace engineering, (though never yet putting that knowledge to use) Now that the parts arrived I have no idea what to do with them. I don't have much hardware or software knowledge and I don't even know how to assemble the flight controller. Anyone know a way I could get that info to get led through, or anyone willing to lead me through it? It would be a huge help.

**Comment by sensei-hacker** [comment]
Two YouTube channels, Experimental Airlines and Flitetest, have step-by-step videos showing you how to build different foam board planes.

Falling With Style (Mr D) and Painless 360 have step-by-step videos for setting up INAV on a plane. There are also other good channels.

**Reply by AerospaceGuy** [reply]
Thanks for the help, I'll make sure to check them out!
